### PR TITLE
ci: move to sn-gh-workflows

### PR DESCRIPTION
**Pull request by niekvanstaveren**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,9 +37,9 @@
     "sense": "nebula sense",
     "spec": "scriptappy-from-jsdoc -c ./spec-configs/props.conf.js",
     "start": "nebula serve",
+    "test:e2e": "playwright test",
```
Comment (haxxmaxx) [inline package.json:40]: the rendering tests became e2e tests?

Comment (niekvanstaveren) [reply package.json:40]: The default for nebula charts is e2e tests https://github.com/qlik-oss/nebula.js/blob/master/commands/create/templates/sn/none/_package.json#L22

We have not really figured out if we need to split them up or not in the script. I think both rendering and integration tests can both live under the test:e2e command

Comment (haxxmaxx) [reply package.json:40]: ok makes sense

**Review by haxxmaxx — APPROVED** [review]

**Comment by niekvanstaveren** [comment]
We can only fully move when sn-gh-workflows support API governance through the release process. Will come back to this after my holiday

**Pull request merged**